grandnode/grandnode
# Handle image converting errors

Type: **bugfix**

## Issue
For now, when you convert images to webp format on Maintenance page and get error on one of pictures - your application crash. I can not found "invalid" image because no logs exists and images to many for manual investigation.

## Solution
If some image conversion get error we log error with picture id and continue process other images.

## Breaking changes
No.

## Testing
1. Upload some not webp images
2. Start convert images on Maintenance page
3. If some broken image exists see Logs page
